modular_zubbers/code/modules/mob/check_pulse.dm

```diff
@@ -0,0 +1,27 @@
+/mob/verb/check_pulse(atom/A as mob in view())
+	set name = "Check Pulse"
+	set category = "Object"
+
+	DEFAULT_QUEUE_OR_CALL_VERB(VERB_CALLBACK(src, PROC_REF(_check_pulse), A))
+
+
+/mob/proc/_check_pulse(mob/atomic)
+	if(client && !(atomic in view(client.view, src)) && !client in orange(1, atomic))
+		return FALSE
+	balloon_alert(client.mob, "checking pulse...")
+	if(do_after(client.mob, 4 SECONDS, atomic))
+		if(isliving(client.mob))
+			to_chat(atomic, span_userdanger("You feel [client.mob] grasp you, feeling you closely!"))
```

Comment:
Extremely ambiguous (`You feel [client.mob] grasp your wrist, pressing their fingers into it!` or similar would work better), and no need for it to be a userdanger as it isn't a combat action. A warning would be fine.

Reply:
I tried to write it in a way that didn't assume anything about the person being checked eg. having arms, limbs, a head even